### PR TITLE
Graphics window

### PR DESCRIPTION
**Pull request by MatthewMeacham**
Added the graphics window/Konva to the run component.

Still will need to add the graphics-window component to the debug component, but that can hold off until the design/structure of this is approved.

Also this request is to merge into shaminoo/SmallBasic-Online/master, should I instead be requesting a merge into the main one (Omar's)?

### CODE REVIEW

File: src/app/components/common/graphics-window/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    this.state = {};
+  }
+
+  public componentDidMount(): void {
+    this.tokens = [];
+
+    const stage = new Konva.Stage({
+      container: "graphics-container",
+      width: document.getElementById("graphics-container")!.offsetWidth,
```
Comment (allisoninman) [inline src/app/components/common/graphics-window/index.tsx:31]: I know we were unsure, is there anyone who can advise on if this is the best way to get this? @shaminoo ?

File: src/app/components/common/graphics-window/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      height: document.getElementById("graphics-container")!.offsetHeight
+    });
+    const layer = new Konva.Layer();
+    stage.add(layer);
+
+    this.setState({
+      ...this.state,
```
Comment (allisoninman) [inline src/app/components/common/graphics-window/index.tsx:38]: What are the dots?

File: src/app/components/common/graphics-window/index.tsx (hunk 1 above)
Comment (MatthewMeacham) [reply src/app/components/common/graphics-window/index.tsx:31]: https://reactjs.org/docs/refs-and-the-dom.html

**Comment by allisoninman** [comment]
Works great!

**Review by shaminoo — APPROVED** [review]

**Review by allisoninman — APPROVED** [review]

**Pull request merged**